healsparse/geom.py

```diff
@@ -0,0 +1,668 @@
+import numpy as np
+import healpy as hp
+from .healSparseMap import HealSparseMap
+
+
+def make_circles(*, ra, dec, radius, value):
+    """
+    make multiple Circles
```

Comment:
I think this should have additional doc string to give a particular use case as an example.

Reply:
done